nixos/modules/services/audio/mpd.nix

```diff
@@ -120,13 +120,29 @@ in {
       user = mkOption {
         type = types.str;
         default = name;
-        description = "User account under which MPD runs.";
+        description = ''
+          User account under which MPD runs.
+
+          <note><para>
+          If left as the default value this user will automatically be created
+          on system activation, otherwise the sysadmin is responsible for
```

Comment:
```suggestion
          on system activation, otherwise you are responsible for
```

Sysadmin seems a bit too specific, as a configuration can be written by an engineer who is not directly involved in operations.

Reply:
@roberth why is it that the best sysadmins don't like being labelled "sysadmin"? :wink: :smile:

Sounds good. Will implement on next push :+1:

Reply:
Probably because Windows on enduser machines always asked to consult the administrator. Sobering experience.

Reply:
> Probably because Windows on enduser machines always asked to consult the administrator. Sobering experience.

![](https://www.prolved.com/wp-content/uploads/2018/01/UnableToContinue.png)

``I am the administorator! 😠 ``